pkg/cmd/cli/cmd/importimage.go

```diff
@@ -440,16 +463,13 @@ func (o *ImportImageOptions) newImageStream() (*imageapi.ImageStream, *imageapi.
 							Kind: "DockerImage",
 							Name: from,
 						},
+						ReferencePolicy: imageapi.TagReferencePolicy{
```

Comment:
i guess this should not be set at all if the reference policy is not set...

i would check below this and generate the ReferencePolicy only if `len(referencePolicy) > 0`